components/mixins/shared/external-source-system-audit-details.schema.json

```diff
@@ -15,7 +15,8 @@
     "https://ns.adobe.com/xdm/context/account",
     "https://ns.adobe.com/xdm/context/opportunity-contactrole",
     "https://ns.adobe.com/xdm/context/activitylog",
-    "https://ns.adobe.com/xdm/context/profile"
```

Comment:
Remove all entries from array to make it a generic mixin usable by any class. The array is now used to constrain to a specific class(es) only.